source/ui/BioXAS/BioXASSSRLMonochromatorConfigurationView.cpp

```diff
@@ -10,21 +10,6 @@ BioXASSSRLMonochromatorConfigurationView::BioXASSSRLMonochromatorConfigurationVi
 
 	// Create UI elements.
 
-	regionEditor_ = new BioXASSSRLMonochromatorRegionControlEditor(0);
-	regionEditor_->setTitle("Region");
-
-	energyEditor_ = new AMExtendedControlEditor(0);
-	energyEditor_->setTitle("Energy (encoder)");
-	energyEditor_->setControlFormat('f', 2);
-
-	calibrateEnergyButton_ = new QPushButton("Calibrate");
```

Comment:
I guess I just wasn't sure they were being used. Probably a better place to remove some of this functionality would be in the EnergyCalibration pull request (#1473). I'll add this back in here.